laravel-mfa-example/resources/views/factor_detail.blade.php

```diff
@@ -32,7 +42,6 @@
                         </div>
                         <div>
                             <p>Type: <code>{{$factor->type}}</code></p>
-                            <p>Environment: <code>{{$factor->environmentId}}</code></p>
                             @if ($factor->type == "sms") 
                             <p>Phone Number: <code>{{$phone_number}}</code></p>
                             @endif 
```

Comment:
The qr code padding/margin looks off here. Can you add a bit more on the top so it's not touching the title? 
![CleanShot 2022-08-26 at 14 45 40](https://user-images.githubusercontent.com/47123183/186988245-1cf191d2-1d1f-4232-bac4-de37ca5d5ead.png)

Reply:
threw some 8vh padding on the qr class